bundles/com.salesforce.bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/lang/jvm/BazelJvmClasspath.java

```diff
@@ -195,6 +195,8 @@ public BazelJvmClasspathResponse getClasspathEntries() {
                         // java_test aspect should be analyzed for implicit dependencies
                         if ("java_test".equals(jvmTargetInfo.getKind())) {
                             implicitDeps = implicitDependencyHelper.computeImplicitDependencies(bazelWorkspace, jvmTargetInfo);
+                            // there is no need to process test jar further
+                            continue;
```

Comment:
Thanks for the fix, looks right. We really need to refactor this class. It is too hard to reason about each of these code paths